cinch/roles/nginx/handlers/main.yml

```diff
@@ -0,0 +1,8 @@
+- name: reload systemd
```

Comment:
2.4 hasn't even been released yet. However, that is the next version that will come out, so that probably means the current git master of Ansible has support for this.